src/insights_client/tests/test_motd.py

```diff
@@ -0,0 +1,97 @@
+import os
+import tempfile
+
+import unittest
+import unittest.mock
+
+import insights_client
+
+
+class TestMOTD(unittest.TestCase):
+    """Verify the behavior of MOTD file."""
+
+    def setUp(self):
+        self.dir = tempfile.TemporaryDirectory()
```

Comment:
the other unit tests use the pytest API rather than `unittest.TestCase`, so it'd be better for the test cases here to follow the same approach

in that case, the `tmp_path` fixture will give you a new temporary directory for every test in which it is used

Reply:
ACK, done with fixtures instead. Whether it is more or less readable I cannot say.